src/components/WwwFrame/Menus/CustomSecondaryMenu.vue

```diff
@@ -138,7 +133,6 @@ export default {
 
 	.sec-ter-combo-nav {
 		position: absolute;
-		top: rem-calc(90);
```

Comment:
Allows the menu to display correctly, when menu title (for example 'Promotional Terms...' ) is longer than 1 line